# All programs meant to be called by other components of Cherokee (and
# not by a human) should go into /usr/share/cherokee, not /usr/bin
#
Index: cherokee/performance.conf.sample.pre
===================================================================
--- cherokee.orig/performance.conf.sample.pre	2009-01-23 15:18:15.000000000 -0600
+++ cherokee/performance.conf.sample.pre	2009-01-23 15:25:55.000000000 -0600
@@ -13,7 +13,7 @@
 server!keepalive = 1
 server!keepalive_max_requests = 1000
 server!server_tokens = full
-server!panic_action = %prefix%/bin/cherokee-panic
+server!panic_action = %prefix%/share/cherokee/cherokee-panic
 server!pid_file = %localstatedir%/run/cherokee.pid
 
 # Default virtual server
Index: cherokee/admin/configured.py.pre
===================================================================
--- cherokee.orig/admin/configured.py.pre	2008-12-09 15:54:55.000000000 -0600
+++ cherokee/admin/configured.py.pre	2009-01-23 15:25:55.000000000 -0600
@@ -14,7 +14,7 @@
 CHEROKEE_ADMINDIR   = join (PREFIX, "share/cherokee/admin")
 CHEROKEE_ICONSDIR   = join (PREFIX, "share/cherokee/icons")
 CHEROKEE_THEMEDIR   = join (PREFIX, "share/cherokee/themes")
-CHEROKEE_PANIC_PATH = join (PREFIX, "bin/cherokee-panic")
+CHEROKEE_PANIC_PATH = join (PREFIX, "share/cherokee/cherokee-panic")
 CHEROKEE_PLUGINDIR  = join (LIBDIR, "cherokee")
 CHEROKEE_DATADIR    = join (DATADIR, "cherokee")
 CHEROKEE_DEPSDIR    = join (DATADIR, "cherokee/deps")
Index: cherokee/cherokee.conf.sample.pre
===================================================================
--- cherokee.orig/cherokee.conf.sample.pre	2009-01-23 15:25:48.000000000 -0600
+++ cherokee/cherokee.conf.sample.pre	2009-01-23 15:25:55.000000000 -0600
@@ -15,7 +15,7 @@
 server!keepalive = 1
 server!keepalive_max_requests = 500
 server!server_tokens = full
-server!panic_action = %prefix%/bin/cherokee-panic
+server!panic_action = %prefix%/share/cherokee/cherokee-panic
 server!pid_file = %localstatedir%/run/cherokee.pid
 
 # Default virtual server
